server/text_generation_server/models/__init__.py

```diff
@@ -110,6 +111,138 @@
     __all__.append(Mamba)
 
 
+class ModelType(enum.Enum):
+    MAMBA = {
+        "type": "ssm",
+        "name": "Mamba",
+        "url": "https://huggingface.co/state-spaces/mamba-2.8b-slimpj",
+    }
+    GALACTICA = {
```

Comment:
I think this is `opt` as well based on the config file?

Reply:
Probably not. I'm not familiar with the codebase, but we tend to remove a lot of ifs from modeling code, meaning than in TGI, some model types which are the same in transformers , are different here (like gpt_bigcode which has models declared as gpt2 but the modeling is actually quite different).